### ISSUE TITLE
Support running one Eowyn instance that runs tests on multiple stations

### ISSUE THREAD

**Issue by vveralag** [vveralag/eowyn_issues#97]
Created by **Gitlab User gorly**: 
One Eowyn for multiple platforms instead one Eowyn to single platform

**Comment by vveralag** [comment]
Commented by **Gitlab User danielcm**: 
changed due date to December 31, 2021

**Comment by vveralag** [comment]
Commented by **Gitlab User danielcm**: 
Step 1 !138 was merged

**Comment by vveralag** [comment]
Commented by **Gitlab User danielcm**: 
added to epic &1

**Comment by vveralag** [comment]
Commented by **Gitlab User danielcm**: 
changed time estimate to 3w

**Comment by vveralag** [comment]
Commented by **Gitlab User danielcm**: 
Let's talk about Frodo redesign to run in parallel. I'm creating a new issue for that.

**Comment by vveralag** [comment]
Commented by **Gitlab User danielcm**: 
marked this issue as related to #146
